mesmerize_napari/napari1d_manager.py

```diff
@@ -208,6 +211,7 @@ def __init__(self, batch_item: pd.Series):
         self.batch_item = batch_item
         self.viewer = napari.Viewer(title="MCORR Visualization")
 
+        # Load input movie optional: Create checkbox
```

Comment:
you mean for side-by-side? The user can load the input move and mcorr movie and click the grid button to see side-by-side, this is the most "naparic" way to do it.

Reply:
nvm, didn't realize this is a new viewer

Reply:
don't make the checkbox, just load the input movie always. it's a memmap anyways